test/modules/schedule/domain/bloc/schedule_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
```

Comment:
When testing a class file, copy the file name and add the suffix "_test".

Copy the directory as well